docker/action-scripts/build-clion-plugin.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#install java zulu 11 distribution
+sudo apt update -y
+sudo apt install dirmngr --install-recommends -y
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
+sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main' -y
+
+sudo apt update -y
+sudo apt install zulu-11 -y
```

Comment:
Why we need zulu instead of open-jdk?

Reply:
Initially github actions for IJ Plugin use zulu (https://github.com/JetBrains/intellij-platform-plugin-template/blob/8188176d4272378939bfcc6b0bfc2c2008c0d2ca/.github/workflows/build.yml#L52), I think open-jdk can be used instead, but did not check it.

Reply:
Ok but then use same way to install it. Because add by key don't look like good solution for CI

Reply:
fixed